cogs/help.py

```diff
@@ -0,0 +1,167 @@
+from inspect import Parameter
+from typing import Coroutine, Function, Optional
+
+import discord
+from discord.ext import commands, menus
+import discord.utils
+
+from .utils import constants
+
+
+class HelpSource(menus.ListPageSource):
+    def __init__(self, signature: Function, filter_commands: Coroutine, prefix: str, author: discord.User, cogs: dict) -> None:
+        self.get_command_signature = signature
+        self.filter_commands = filter_commands
+        self.prefix = prefix
+        self.menu_author = author
+        super().__init__([(cog, cogs[cog]) for cog in sorted(cogs, key=lambda cog: cog.qualified_name if cog else "ZZ")], per_page=1)
+
+    async def format_page(self, menu: menus.MenuPages, cog_tuple: tuple) -> discord.Embed:
+        cog, commands = cog_tuple
+        embed = discord.Embed(
+            title=f"Help for {cog.qualified_name if cog else 'unclassified commands'}",
+            description=(
+                'Command syntax : `<Those arguments are required>`. `[Those aren\'t]`\n'
+                f'[Everything to know about my glorious self]({constants.invite_link} "Invite link")\n'
+                f'The prefix for this channel is `{self.prefix}`\n{cog.description if cog else ""}'
+            ),
+            colour=0xffff00,
+        )
+        embed.set_author(
+            name=self.menu_author.display_name,
+            icon_url=str(self.menu_author.avatar_url),
```

Comment:
Is it really needed to convert this into string? I think this already is a string on return

Reply:
No, it's an `Asset` object : https://discordpy.readthedocs.io/en/latest/api.html#discord.User.avatar_url